via/templates/error.jinja2

```diff
@@ -0,0 +1,211 @@
+{% set LINK = {
```

Comment:
The filename should be `error.html.jinja2` (that's the naming convention we use everywhere)

Reply:
This is now "exception.html.jinja2"